### ISSUE TITLE
Application Returns to Previous Window on Focus Loss

### ISSUE THREAD

**Issue by davirec** [andstatus/andstatus#595]
Hello,

I've encountered a bug where the application reverts to the previous window when it loses focus. This issue happens when switching between apps or when another program overlaps the application window.

You can easily reproduce the issue by using the "recent apps" or "back" buttons, or by allowing another application to take focus.

Could you please look into this?

<div id="ladoalado" style="display:flex;justify-content:center;">

<figure id="container">Before and after "recent apps" and "back"<br>
<img src="https://github.com/user-attachments/assets/b7766b39-232b-4ce0-ac74-9bbfa06f4d04" width=20% height=20% >
</figure>
<figure id="container">
<img src="https://github.com/user-attachments/assets/8f02e525-d993-4e3c-bcff-58c69e6a575d" width=20% height=20%>
</figure>

</div> 

<div id="ladoalado" style="display:flex;justify-content:center;">

<figure id="container">Before and after "recent apps" and "back"<br>
<img src="https://github.com/user-attachments/assets/b2452f6f-38c5-421d-aebb-c9db0dd8a42e" width=20% height=20% >
</figure>
<figure id="container">
<img src="https://github.com/user-attachments/assets/91efbc91-dd37-4af2-adaf-38ecb3b2300e" width=20% height=20%>
</figure>

</div>

**Comment by yvolk** [comment]
Yes, as I remember, it was very hard to make the Settings screens working "better than out of the box".
So I'm afraid this won't be fixed until migration to some better Settings control...
Or you could try to fix this and  provide a Pull request?!
